static/js/lib/courseAPI.js

```diff
@@ -0,0 +1,18 @@
+import moment from "moment"
```

Comment:
Nitpick: please rename the file from `courseAPI.js` to `courseApi.js` (and `courseAPI_test.js` to `courseApi_test.js`). Our general file naming pattern has been to avoid all-capital acronyms.